.gitignore

```diff
@@ -5,6 +5,7 @@ MANIFEST
 doc/build/*
 logs/*
 tests
+.pytest_cache/
```

Comment:
I'm not convinced delaying it helps, as it can only prevent other contributors (who actually run tests 😉) from accidentally committing a big glob of garbage between now and "some point". There is no downside.

Reply:
It's a principle, that's all. This PR is all about updating the title finder, and I'm a very strong believer in keeping PRs down to one thing. All that said, don't let my stated plan to add some ignores stop you from making your own PR to update `.gitignore`! I'd just rather not touch an unrelated file in this particular pull request. :cat: